### PR TITLE
created video component with text overlay - additional CSS

### PR DESCRIPTION
**Pull request by DeagoBrowne**
also has original about page that I created

### CODE REVIEW

**Review by HThanki — COMMENTED** [review]
Hi @DeagoBrowne, good work. Please can we move the video component to a client/src/components/Video folder. Otherwise looks good. Thanks

**Comment by robota-x** [comment]
> Hi @DeagoBrowne, good work. Please can we move the video component to a client/src/components/Video folder. Otherwise looks good. Thanks

Let's defer this for a future PR for now. Similar to my comment here about hardcoded values Hi @DeagoBrowne, good work. Please can we move the video component to a client/src/components/Video folder. Otherwise looks good. Thanks

**Review by robota-x — APPROVED** [review]

File: client/src/Video.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const Video = () => {
+  return (
+    <div className="videoWithOverlay">
+      <video className="video" src="https://cdn.pixabay.com/vimeo/636709154/Skyscrapers%20-%2091744.mp4?width=2560&expiry=1657038723&hash=256e8f29b8d7ec5a61fff7e89d7626efb621e350"
```
Comment (Monika39-maker) [inline client/src/Video.js]: I still cant see any video rendered in the page. Can you please have a look and find why?

Comment (DeagoBrowne) [reply client/src/Video.js]: Video Serving Issue - replaced video with placeholder image for now.

**Pull request merged**